apps/demo-react/components/Wagmi.tsx

```diff
@@ -0,0 +1,31 @@
+import { WagmiConfig, createClient, configureChains } from 'wagmi';
+import { infuraProvider } from 'wagmi/providers/infura';
+import { alchemyProvider } from 'wagmi/providers/alchemy';
+import { jsonRpcProvider } from 'wagmi/providers/jsonRpc';
+import { publicProvider } from 'wagmi/providers/public';
+import { ReactNode } from 'react';
+import { mainnet, goerli } from 'wagmi/chains';
+
+const { provider, webSocketProvider } = configureChains(
+  [mainnet, goerli],
+  [
+    infuraProvider({ apiKey: '8922a14a760246d7b9b415961f8cd01c', priority: 0 }),
```

Comment:
Is it ok that these keys are public? Maybe it makes sense to move them to .env? even if they are your personal free keys